main.py

```diff
@@ -108,18 +108,19 @@ def setup_buttons(self):
         row = 0
         col = 0
         for button in self.rocket.buttons.keys():
-            exec(f"self.{button}Button = QtWidgets.QPushButton(self.centralwidget)")
+            setattr(self, button + "Button", QtWidgets.QPushButton(self.centralwidget))
+            qt_button = getattr(self, button + "Button")
             sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Minimum)
             sizePolicy.setHorizontalStretch(0)
             sizePolicy.setVerticalStretch(0)
-            exec(f"sizePolicy.setHeightForWidth(self.{button}Button.sizePolicy().hasHeightForWidth())")
-            exec(f"self.{button}Button.setSizePolicy(sizePolicy)")
+            sizePolicy.setHeightForWidth(getattr(self, button + "Button").sizePolicy().hasHeightForWidth())
+            qt_button.setSizePolicy(sizePolicy)
             font = QtGui.QFont()
             font.setPointSize(35)
             font.setKerning(True)
-            exec(f"self.{button}Button.setFont(font)")
-            exec(f"self.{button}Button.setObjectName('{button}Button')")
-            exec(f"self.gridLayout_5.addWidget(self.{button}Button, {row}, {col}, 1, 1)")
+            qt_button.setFont(font)
+            qt_button.setObjectName(f"{button}Button")
+            self.gridLayout_5.addWidget(qt_button, row, col, 1, 1)
```

Comment:
Wow looks great! No execs and its all actual code 🚀 

I'm assuming it all works just fine?

Reply:
Everything works as before.